doc/man3/OpenSSL_version.pod

```diff
@@ -137,8 +137,8 @@ The current OpenSSL cpu settings.
 This is the current setting of the cpu capability flags. It is usually
 automatically configured but may be set via an environment variable.
 The value has the same syntax as the environment variable.
-For x86 the string looks like C<CPUINFO: OPENSSL_ia32cap=0x123:0x456>.
-Or C<CPUINFO: N/A> if not available, such as when configured with B<no-asm>.
+For x86 the string looks like C<CPUINFO: OPENSSL_ia32cap=0x123:0x456>
+or C<CPUINFO: N/A> if not available, e.g. no-asm build.
```

Comment:
Skip the e.g. bit entirely. It's meaningless to those who do not build openssl on their own (I know that there are other places where configuration stuff is mentioned, and that might be worth scrutiny, but not in this PR)

Reply:
okay.